src/dpq2/conv/to_d_types.d

```diff
@@ -178,20 +179,46 @@ if( is( T == Json ) )
 public void _integration_test( string connParam ) @system
 {
     auto conn = new Connection(connParam);
+    conn.exec("set timezone to 0"); // to return times in UTC and so make the test reproducible in databases with other TZ
```

Comment:
Probably, isn't need if assumptions about TZ will be removed?

Reply:
This is due to this test:
```D
C!SysTime(SysTime(DateTime(1997, 12, 17, 7, 37, 16), dur!"usecs"(12), new immutable SimpleTimeZone(2.dur!"hours")), "timestamp with time zone", "'1997-12-17 05:37:16.000012+00'");
```
Because without specific TZ PG will convert the value to my local TZ which will work for me, but not for anyone in different TZ.
For example I'll get '1997-12-17 06:37:16.000012+01' then.

Reply:
Maybe better, as exclusion, generate '1997-12-17 05:37:16.000012+00' string with addition of your local TZ?
Because `conn.exec("set timezone to 0");` theoretically can mask some future TZ-related bugs with TZ assumptions.

Reply:
Ok, I've reworked this.
It was more complicated than I thought as SysTime.toISOExtString with time in LocalTime does not output the UTC offset so I had to append it myself to generate expected output from PG.

Note that if the integration tests DB is set to use other than LocalTime TZ, test won't work.
For that we would have to check TZ from the DB.